geoips/commandline/test_interfaces.py

```diff
@@ -21,9 +21,12 @@
 from geoips import interfaces
 from geoips.interfaces.base import BaseInterface
 import sys
+import logging
+
 
 FAILED_INTERFACE_HEADER_PRE = "FAILED: issues found within "
 FAILED_INTERFACE_HEADER_POST = "interface:"
+LOG = logging.getLogger(__name__)
```

Comment:
This one also probably needs to call setup_logging, right?

Reply:
Done.